src/main/resources/pdiresources/transformations/transformation-for-encounter.ktr

```diff
@@ -0,0 +1,1083 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please rename the transformation to `merge-encounter`.

Reply:
done